### PR TITLE
RgbToYiq and YiqToRgb operations

### PR DESCRIPTION
**Pull request by quickwritereader**
Cpu implementation and Tests.

### CODE REVIEW

File: libnd4j/include/ops/declarable/generic/images/yiqToRgb.cpp [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#include <ops/declarable/CustomOperations.h>  
+#include <helpers/ConstantTadHelper.h>
+#include <execution/Threads.h>
+
+namespace nd4j {
+    namespace ops {
+
+
+
+        CONFIGURABLE_OP_IMPL(yiq_to_rgb, 1, 1, false, 0, 0) {
```
Comment (raver119) [inline libnd4j/include/ops/declarable/generic/images/yiqToRgb.cpp]: This op clearly can be called "inplace" by the looks ok if, so let's allow that

**Review by raver119 — CHANGES_REQUESTED** [review]
LGTM, but:

1) One minor tweak required for inplace execution allowance.
2) CUDA version to be added, I'll add it in this branch.

**Comment by raver119** [comment]
Also, please stop reusing the same branch over and over again.

**Comment by saudet** [comment]
> Also, please stop reusing the same branch over and over again.

Why is that a bad thing? We can think of it as a "devel" branch

**Comment by raver119** [comment]
If you have branch used and stored in CLion/Intellij it manages to store previous version of the branch, and introduces conflicts that do not exist in reality.

**Comment by quickwritereader** [comment]
ok. got my lesson. will use issue named branch names

**Comment by saudet** [comment]
> If you have branch used and stored in CLion/Intellij it manages to store previous version of the branch, and introduces conflicts that do not exist in reality.

That's probably just a matter of deleting your local branch with `git branch -D` or something. Unless you want to leave them around for some reason...

**Pull request merged**